src/Altinn.Notifications/Validators/Rules/EmailTemplateRules.cs

```diff
@@ -76,9 +77,17 @@ private static void ApplyCommonEmailTemplateRules<T>(IRuleBuilderInitial<T, Emai
                         .NotEmpty()
                         .WithMessage("The email template body must not be empty.");
 
+                    template.RuleFor(t => t.Body)
```

Comment:
it's possible to chain the rules if they pertain to the same property e.g.

```
template.RuleFor(t => t.Body)
	.NotEmpty()
	.WithMessage("The email template body must not be empty.")
	.Must(e => e.DoesNotContainUrl())
	.WithMessage("The email template body must not contain any URLs.");
```

I see pro's and cons with both approaches, so just wanted to mention it

Reply:
We use few validation rules, so chaining them together is ok.